【发现已存在的游戏窗口】

【现在处于主菜单】
[WTG]images/battery.png confident：1.0
Using CPU. Note: This module is much faster with a GPU.
当前电量为：180 
任务开始，执行 1 次循环

【现在处于主菜单】
[WTG]images/lobby.png confident：0.9999198317527771
[WTG]images/tactics.png confident：0.9765910506248474
[WTG]等待目标中 images/train.png
[WTG]等待目标中 images/train.png
[WTG]等待目标中 images/train.png
[WTG]等待目标中 images/train.png
[WTG]等待目标中 images/train.png
[WTG]等待目标中 images/train.png
[WTG]等待目标中 images/train.png
-------------------------------------------------
Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 748, in run
    TASK()
    ~~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 689, in <lambda>
    TASK=lambda :upgrade(task_list[task_index])
                 ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 524, in upgrade
    click('images/train.png')
    ~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 82, in click
    match_tpl(tpl,'click',skip=skip)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 80, in match_tpl
    time.sleep(wait_speed)
    ~~~~~~~~~~^^^^^^^^^^^^
KeyboardInterrupt

Exception:主线程出错
按回车键退出(Enter)Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 748, in run
    TASK()
    ~~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 689, in <lambda>
    TASK=lambda :upgrade(task_list[task_index])
                 ~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 524, in upgrade
    click('images/train.png')
    ~~~~~^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 82, in click
    match_tpl(tpl,'click',skip=skip)
    ~~~~~~~~~^^^^^^^^^^^^^^^^^^^^^^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 80, in match_tpl
    time.sleep(wait_speed)
    ~~~~~~~~~~^^^^^^^^^^^^
KeyboardInterrupt

During handling of the above exception, another exception occurred:

Traceback (most recent call last):
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 774, in <module>
    run()
    ~~~^^
  File "C:\Users\gelon\Desktop\Pytutorial\ZenlessZoneZero_Automation\ZA_Main.py", line 771, in run
    input('Exception:主线程出错\n按回车键退出(Enter)')
    ~~~~~^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "<frozen codecs>", line 325, in decode
UnicodeDecodeError: 'utf-8' codec can't decode byte 0xff in position 0: invalid start byte
